src/components/Clients/ClientListTableRow.jsx

```diff
@@ -65,45 +62,55 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
   };
 
   return (
-    <StyledTableRow>
-      <StyledTableCell align="center">
-        <Checkbox
-          id={labelId}
-          color="primary"
-          checked={selectedUser.webId === client.webId}
-          inputProps={{
-            'aria-labelledby': labelId
-          }}
-          onClick={() => {
-            handleSelectClient(client);
-          }}
-        />
-      </StyledTableCell>
-      <StyledTableCell align="center">{client.person}</StyledTableCell>
-      {/* ***** TODO: Switch this webId to being a small Notes section */}
-      {/* seems having a link or even displaying another user's pod is completely useless/irrelevant */}
-      {/* see no reason it would ever need to be used, but notes/comments will be of utmost importance to caseworkers */}
-      <StyledTableCell align="center">
-        <Link
-          to={client.webId}
-          target="_blank"
-          rel="noreferrer"
-          style={{ textDecoration: 'none', color: theme.palette.primary.dark }}
-        >
-          Link to Pod Profile
-        </Link>
-      </StyledTableCell>
-      <StyledTableCell align="center">
-        <IconButton size="large" edge="end" onClick={handlePinClick}>
-          {pinnedIcon}
-        </IconButton>
-      </StyledTableCell>
-      <StyledTableCell align="center">
-        <IconButton size="large" edge="end" onClick={() => handleDeleteClient()}>
-          <DeleteOutlineOutlinedIcon />
-        </IconButton>
-      </StyledTableCell>
-    </StyledTableRow>
+    <>
```

Comment:
React components can only return one thing, so it was necessary because I was returning the `<StyledTableRow>` and the `<DeleteClientModal>`.  The fragments don't produce an actual dom element so it's not in the user facing code, just here in the dev side of things.

But with all that said, could put the modal inside the table row so no need for react fragments, but even though it would work fine I felt like this is a separate component from the table row so I separated it. 

I'm up for switching it, if you all feel like it should be switched no problem.

Reply:
We do have two separate components in here after all (one for the row, one for the modal). I think wrapping this in a fragment is appropriate.